### DISCUSSION TITLE
post/asn-2-bgp-session/

### DISCUSSION THREAD

**Discussion by giscus** [lyc8503/lyc8503#27] [Announcements]
# post/asn-2-bgp-session/

前言上一篇 Blog 中我们已经请 LIR 帮我们注册好了自己的 ASN, 并租用(或获赠)了一段 IPv6 地址. 这篇文章中, 我们要让全世界接入互联网的计算机都能访问到这段 IP 地址. BGP 会话理论上, 我们要通过 BGP 协议与其他 AS 内的边界网关连接, 建立 BGP 会话 (BGP Session), 互相交换自己的 IP 段. 但作为新晋的迷你”运营商”, 我们并没有任何物理

https://blog.lyc8503.net/post/asn-2-bgp-session/

**Comment by darwinchow** [comment]
我俩的ASN竟然只差了一位数🫡

**Reply by lyc8503** [reply]
看来申请时间比较接近😂

**Comment by Guochengjie** [comment]
一次性宣告/40，太奢侈了.....我都分48

**Reply by lyc8503** [reply]
第五 part 我改成宣告 /48 了, 从另一个角度来说, 只是我太穷了只买得起一台服务器, 所以就一把梭全宣告了🥲

**Reply by Guochengjie** [reply]
AS214775 has not been visible in the global routing table since December 06, 2024

为啥你一个路由都没有（（

**Reply by lyc8503** [reply]
> AS214775 has not been visible in the global routing table since December 06, 2024
> 
> 为啥你一个路由都没有（（

~~因为服务器过期了, 暂时没别的用途就没续费~~

~~你看我就说我是真穷吧~~

**Reply by Guochengjie** [reply]
> > AS214775 has not been visible in the global routing table since December 06, 2024
> > 为啥你一个路由都没有（（
> 
> ~因为服务器过期了, 暂时没别的用途就没续费~
> 
> ~你看我就说我是真穷吧~

考虑下vultr吧，那玩意是入门必备...buyvm那边也便宜的..

**Reply by lyc8503** [reply]
> 考虑下vultr吧，那玩意是入门必备...buyvm那边也便宜的..

Vultr 我用过了, 之后的博客也提到过. ~~穷是开玩笑的~~, 价格不是关键因素, 主要是我不太喜欢零零散散开着一大堆闲置的 VPS, 我也没打算长时间维护互联网上的 ASN, 没这个需求. 如果之后我还要尝试 BGP 我就去 DN42 了. 如果某天我去开 IDC 了我再申请个 ASN 吧.🤣

**Reply by Guochengjie** [reply]
原来如此，我确实闲置了一大堆...不过很多还是都是基于需求+特地找了能BGP的服务器

**Comment by Lorien95** [comment]
老师你好，想请教一个问题，我搜了很多地方还是不理解，就是提供托管服务，一般都会给10m，100m带宽，但是我看ip transit有1g或者10g的，这两个是一回事吗，假如我的服务器和ip transit提供商在同一机房，是不是就可以直接买ip transit连接到互联网，而不用托管服务里很小的带宽。

**Reply by lyc8503** [reply]
理论上是可以的，这两个带宽都是通向互联网的带宽，你可以直接购买 IP Transit。服务商给你提供的带宽也是他购买了 IP + IP Transit 后打包再卖给你的。

不过就海外机器来说，很多机器/机房都有 1Gbps 的带宽，IP Transit 的价格应该不会比直接购买带宽低（事实上"优质"线路的 IP Transit 通常比托管服务的带宽贵不少），你购买前可以向商家咨询清楚，是否有流量限制，是否是共享带宽等。

国内机房和服务商我没怎么接触，不太了解。

**Reply by Lorien95** [reply]
终于明白了，感谢╰(*´︶`*)╯

**Comment by justruiqwq** [comment]
大佬peer吗

**Reply by lyc8503** [reply]
暂时没怎么维护，就不单独 peer 了😶

**Reply by justruiqwq** [reply]
大佬你是用v2ray连接到服务器上再用自己的ip的吗，我现在用v2ray,虽然指定了出口，但变成仅ipv6上网了，求解如何让bird的ip优先

**Reply by lyc8503** [reply]
是的，这个可以参考一下 v2 的文档配置，我不用 v2 客户端。

**Comment by justruiqwq** [comment]
大佬有免费的IP Transit推荐吗 <del>想白嫖了</del>

**Comment by MCKero6423** [comment]
能给我分配一个IPV6地址吗？
